.travis.yml

```diff
@@ -102,6 +102,16 @@ jobs:
         RUN_FUNCTIONAL_TESTS=false # Disabled for now. TODO identify suppressions or exclude specific tests
         GOAL="install"
         BITCOIN_CONFIG="--enable-zmq --with-incompatible-bdb --with-gui=qt5 CPPFLAGS=-DDEBUG_LOCKORDER --with-sanitizers=thread --disable-hardening --disable-asm CC=clang CXX=clang++"
+# x86_64 Linux (xenial, no depends, only system libs, sanitizers: address (ASAN))
```

Comment:
Could this be combined with the bionic sanitizer build below instead (assuming that the tests pass)